app.py

```diff
@@ -208,6 +212,32 @@ async def inference_request():
         return jsonify(["InferenceRequestError: " + str(error)]), 400
 
 
+@app.get("/seed-info/<seed_name>")
+async def get_seed_info(seed_name):
+    """
+    Retrieves JSON document containing seed Fact Sheet information
+    """
+    global seed_cache
+    # Verify if seed already in cache
+    if seed_name in seed_cache:  
+        return jsonify(seed_cache[seed_name])
+    
+    try:
+        with urllib.request.urlopen(seed_info_url) as response:
+            result = response.read()
+            result_json = json.loads(result.decode("utf-8"))
+
+            # Cache the result
+            seed_cache = result_json
+
+            return jsonify(result_json[seed_name])
+
+    except urllib.error.HTTPError as error:
+        return jsonify({"error": f"Failed to retrieve the JSON. \
+                        HTTP Status Code: {error.code}"}), 400
+    except Exception as e:
+        return jsonify({"error": str(e)}), 500
+
 @app.get("/ping")
 async def ping():
```

Comment:
I think we've settled on /health:

https://github.com/ai-cfia/membrane-backend/blob/22bf76f70379b7ca6f6534310bb3dece9d5b10e1/app.py#L34

to allow us to put a bit more info on that endpoint in the future.